scripts/resources.js

```diff
@@ -0,0 +1,297 @@
+const $ = require('shelljs')
+const program = require('commander')
+const CONF = require('./local.config')()
+
+// die on first error
+$.config.fatal = true
+
+// VERBOSE=1 node deploy.js
+$.config.verbose = !!process.env.VERBOSE || false
```

Comment:
there are two problems with this statement :)

`!!"0" === true`

also the `|| false` is unnecessary (`A || false === A`)